sql/firefox_nondesktop_exact_mau28_by_product.sql

```diff
@@ -9,9 +9,8 @@ SELECT
   SUM(IF(country IN ('US', 'FR', 'DE', 'UK', 'CA'), wau, 0)) AS tier1_wau,
   SUM(IF(country IN ('US', 'FR', 'DE', 'UK', 'CA'), dau, 0)) AS tier1_dau
 FROM
-  firefox_nondesktop_exact_mau28_by_product_by_dimensions_v1
-WHERE
-  submission_date = @submission_date
+  firefox_nondesktop_exact_mau28_by_product_by_dimensions
```

Comment:
I think the tables should be `firefox_nondesktop_exact_mau28_by_dimensions` and `firefox_nondesktop_exact_mau28_by_product`